tasks/change_config.yml

```diff
@@ -22,10 +22,10 @@
 - name: Wait for all pods to be ready again
   ansible.builtin.shell: |
     set -o pipefail
-    {{ rke2_data_path }}/bin/kubectl --kubeconfig /etc/rancher/rke2/rke2.yaml get pods -A --field-selector=metadata.namespace!=kube-system | grep -iE "crash|error|init|terminating" | wc -l
+    {{ rke2_data_path }}/bin/kubectl --kubeconfig /etc/rancher/rke2/rke2.yaml get pods -A --field-selector=metadata.namespace!=kube-system,status.phase!=Running,status.phase!=Succeeded --ignore-not-found | wc -l
   args:
     executable: /bin/bash
-  failed_when: "all_pods_ready.rc not in [ 0, 1 ]"
+  failed_when: "all_pods_ready.rc != 0"
```

Comment:
I am not sure why return code 1 was considered ok here, so i made this change, if there is something im not seeing, please comment @MonolithProjects

Reply:
It was like that because if `grep` found no matches (all pods in Ready state), the command would return `1` and the task would fail. But with your approach it's fine to change this. I guess now you can even remove this line.

Reply:
Removed